executor/builder.go

```diff
@@ -1197,6 +1198,24 @@ func (b *executorBuilder) buildExplain(v *plannercore.Explain) Executor {
 		if b.ctx.GetSessionVars().StmtCtx.RuntimeStatsColl == nil {
 			b.ctx.GetSessionVars().StmtCtx.RuntimeStatsColl = execdetails.NewRuntimeStatsColl(nil)
 		}
+		// If the resource group name is not empty or "default", we could collect and display the RU
+		// runtime stats for analyze executor.
+		resourceGroupName := b.ctx.GetSessionVars().ResourceGroupName
+		if len(resourceGroupName) > 0 && resourceGroupName != ddl.DefaultResourceGroupName {
```

Comment:
BTW, need check the variable`tidb_enable_resource_control`?

Reply:
If the switch is not on, the user won't be able to bind the resource group. So I think checking the resource group name is enough.